### PR TITLE
beginner_source/nlp/pytorch_tutorial.py 번역

### PR DESCRIPTION
**Pull request by 2kkeullim**
beginner_source/nlp/pytorch_tutorial.py 번역

## 라이선스 동의
_변경해주시는 내용에 BSD 3항 라이선스가 적용됨을 동의해주셔야 합니다._<br />
_더 자세한 내용은 [기여하기 문서](https://github.com/PyTorchKorea/tutorials-kr/blob/master/CONTRIBUTING.md)를 참고해주세요._<br />
_동의하시면 아래 `[ ]`를 `[x]`로 만들어주세요._<br />

- [x] 기여하기 문서를 확인하였으며, 본 PR 내용에 BSD 3항 라이선스가 적용됨에 동의합니다.

## 관련 이슈 번호
_이 Pull Request와 관련있는 이슈 번호를 적어주세요._<br />
_이슈 또는 PR 번호 앞에 #을 붙이시면 제목을 바로 확인하실 수 있습니다. (예. #999 )_

- **이슈 번호**: #570 

## PR 종류
_이 PR에 해당되는 종류 앞의 `[ ]`을 `[x]`로 변경해주세요._<br />
- [ ] 오탈자를 수정하거나 번역을 개선하는 기여
- [x] 번역되지 않은 튜토리얼을 번역하는 기여
- [ ] 공식 튜토리얼 내용을 반영하는 기여
- [ ] 위 종류에 포함되지 않는 기여

## PR 설명
_이 PR로 무엇이 달라지는지 대략적으로 알려주세요._

번역되지 않은 문서를 번역했습니다.
원본 문서 주석 부분에 restructuredtext 문법으로 작성해
restructuredtext 문법이 그대로 출력되는 문제가 있긴 하지만
일단은 원본 그대로 두었습니다.

### CODE REVIEW

File: beginner_source/nlp/pytorch_tutorial.py [hunk 1]
```diff
@@ -19,78 +20,78 @@
...
 T = torch.tensor(T_data)
 print(T)
 
 
 ######################################################################
-# What is a 3D tensor anyway? Think about it like this. If you have a
-# vector, indexing into the vector gives you a scalar. If you have a
-# matrix, indexing into the matrix gives you a vector. If you have a 3D
-# tensor, then indexing into the tensor gives you a matrix!
+# 그건 그렇고 3D tensor는 무엇일까요? 이렇게 생각해 보세요.
```
Comment (kimtaehyeong) [inline beginner_source/nlp/pytorch_tutorial.py:48]: 그건 그렇고 라는 표현보다 그냥 3D tensor는 무엇일까요? 라고 번역하는게 어떤가요?

File: beginner_source/nlp/pytorch_tutorial.py [hunk 2]
```diff
@@ -99,179 +100,178 @@
...
-print(x.view(2, 12))  # Reshape to 2 rows, 12 columns
-# Same as above.  If one of the dimensions is -1, its size can be inferred
+print(x.view(2, 12))  # 2열 12행으로 구조 바꾸기
+# 위와 같습니다.  차원 중 하나가 -1인 경우 그 크기를 유추할 수 있습니다
 print(x.view(2, -1))
 
 
 ######################################################################
-# Computation Graphs and Automatic Differentiation
+# 계산 그래프(Computation Graph) 와 자동미분(Automatic Differentiation)
```
Comment (kimtaehyeong) [inline beginner_source/nlp/pytorch_tutorial.py:144]: 자동 미분 띄어쓰기가 들어가야 될 것 같습니다.

**Review by kimtaehyeong — COMMENTED** [review]
잘 번역해주셨습니다. 크게 두 군데 확인했습니다.

File: beginner_source/nlp/pytorch_tutorial.py (hunk 2 above)
Comment (2kkeullim) [reply beginner_source/nlp/pytorch_tutorial.py]: 반영했습니다! 감사합니당

File: beginner_source/nlp/pytorch_tutorial.py (hunk 1 above)
Comment (2kkeullim) [reply beginner_source/nlp/pytorch_tutorial.py:48]: anyway 의 느낌을 살리고 싶어 그건 그렇고라고 번역했는데 이상한가요??

**Comment by kil-jung-keong** [comment]
번역 잘 해주신것 같습니다. 크게 수정하실 부분은 없어보입니다.

**Review by 95hj — APPROVED** [review]
LGTM!

File: beginner_source/nlp/pytorch_tutorial.py [hunk 3]
```diff
@@ -99,179 +100,178 @@
...
 #
-# The concept of a computation graph is essential to efficient deep
-# learning programming, because it allows you to not have to write the
-# back propagation gradients yourself. A computation graph is simply a
-# specification of how your data is combined to give you the output. Since
-# the graph totally specifies what parameters were involved with which
-# operations, it contains enough information to compute derivatives. This
-# probably sounds vague, so let's see what is going on using the
-# fundamental flag ``requires_grad``.
+# 계산 그래프의 개념은 직접 역전파 변화도(gradient)를 쓰지 않아도 되도록 하여
```
Comment (hyoyoung) [inline beginner_source/nlp/pytorch_tutorial.py:147]: '쓰지 않아도 되도록 하여'는 문장이 조금 어색해보입니다.

계산 그래프의 개념은 직접 역전파 변화도(gradient)를 쓸 필요가 없게 해주며

File: beginner_source/nlp/pytorch_tutorial.py [hunk 4]
```diff
@@ -99,179 +100,178 @@
...
-# First, think from a programmers perspective. What is stored in the
-# torch.Tensor objects we were creating above? Obviously the data and the
-# shape, and maybe a few other things. But when we added two tensors
-# together, we got an output tensor. All this output tensor knows is its
-# data and shape. It has no idea that it was the sum of two other tensors
-# (it could have been read in from a file, it could be the result of some
-# other operation, etc.)
+# 먼저 프로그래머의 관점에서 생각해봅시다. 위에서 우리가 만든
+# torch.Tensor 오브젝트에는 무엇이 저장되어 있을까요? 분명히 데이터와
+# 구조, 그리고 몇 가지 다른 것들이 있을 겁니다. 하지만 2개의 tensor를
```
Comment (hyoyoung) [inline beginner_source/nlp/pytorch_tutorial.py:158]: A, B 그리고 C 라면
번역시에는 그리고는 빼도 좋을듯합니다
A, B, C로 써도 번역시에는 쉽게 읽힙니다

File: beginner_source/nlp/pytorch_tutorial.py [hunk 5]
```diff
@@ -99,179 +100,178 @@
...
+# 하지만, z는 뭔가를 더 알고 있습니다.
 print(z.grad_fn)
 
 
 ######################################################################
-# So Tensors know what created them. z knows that it wasn't read in from
-# a file, it wasn't the result of a multiplication or exponential or
-# whatever. And if you keep following z.grad_fn, you will find yourself at
-# x and y.
+# 즉 Tensor는 무엇이 그들을 만들었는지 압니다. z는 Tensor가 파일에서
```
Comment (hyoyoung) [inline beginner_source/nlp/pytorch_tutorial.py:182]: 즉 Tensor는 무엇이 그들을 만들었는지 압니다

즉 Tensor는 무엇으로 그들이 만들어졌는지 압니다.

**Review by hyoyoung — CHANGES_REQUESTED** [review]
긴 문서를 매끄럽게 번역하느라 고생 많으셨습니다
몇가지 수정사항 부탁드립니다

**Review by hyoyoung — APPROVED** [review]
good
